crates/sui-core/src/epoch/randomness.rs

```diff
@@ -284,8 +285,8 @@ impl RandomnessManager {
 
     /// Processes all received messages and advances the randomness DKG state machine when possible,
     /// sending out a dkg::Confirmation and generating final output.
-    pub fn advance_dkg(&self, batch: &mut DBBatch) -> SuiResult {
-        self.inner.lock().unwrap().advance_dkg(batch)
+    pub fn advance_dkg(&self, batch: &mut DBBatch, round: Round) -> SuiResult {
+        self.inner.lock().unwrap().advance_dkg(batch, round)
```

Comment:
nit: you can use parking_lot::Mutex and then you don't need unwrap()

Reply:
future PR